.github/renovate.json5

```diff
@@ -341,8 +341,8 @@
     },
     {
       "fileMatch": [
-	"^test/kubernetes-test\\.sh$",
```

Comment:
nit: is this intentional?

Reply:
Yes, those two lines had tabs as indent which was inconsistent with the rest of the file, so I replaced them with spaces on the way.